cmd/performance-profile-creator/cmd/root.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"os"
+
+	"k8s.io/klog"
+	"k8s.io/utils/pointer"
+
+	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
```

Comment:
do we really really really need globals?

Reply:
Good point, I can move this to init.

Reply:
We should use local variables as much as we can (possibly packed into a struct). Not sure if moving them to init will work. They can be local in `init()` can work. I was thinking to something like https://github.com/openshift-kni/debug-tools/blob/main/cmd/knit/cmd/root.go - note we still use a global struct, which is still bad. I Didn't have time to check how to make it local to the `root` command and pass it to subcommands.

If we don't plan to use subcommands (for now) we can do like https://github.com/openshift-kni/debug-tools/blob/main/cmd/knit/cmd/cpuaff.go#L35

any way, the less (and more compact, e.g. at least packed into a struct) globals we use, the better.

Reply:
Ack, will update to a struct for now.